docs/source/howto/usage/singularity.rst

```diff
@@ -0,0 +1,17 @@
+The pyAFQ singularity image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Assuming the singularity image ``pyafq_lastest.simg`` exists, trigger the
+entrypoint ``pyafq`` to run the workflow with::
+
+    singularity run \
+        --cleanenv \
+        --bind bids_dir:bids_dir \
+        pyafq_latest.simg bids_dir/config.toml
+
+.. note::
+
+    It may be necessary to set the global variable TEMPLATEFLOW_HOME to
+    reference a pull of `TemplateFlow <https://www.templateflow.org>`_:
+
+    ``export TEMPLATEFLOW_HOME=/path/to/templateflow``
```

Comment:
I was able to execute via apptainer via `$ apptainer run ...`

Reply:
Could you use `apptainer` instead of `singularity` then?

Reply:
Done: 1a04bf59fdff819835b53e98f35daa19306c16e3